src/aleph/handlers/content/vm.py

```diff
@@ -289,6 +301,50 @@ class HasParent(Protocol):
             )
 
 
+def get_extra_storage(content: InstanceContent, session: DbSession) -> int:
+    volumes: int = 0
```

Comment:
Rename to `total_volume_size`. It would be better if this function returned the size in bytes, and then let the caller do whatever it wants (convert to MB, MiB, Watts, I don't care).